### PR TITLE
[HttpKernel] Skip ContainerAwareInterface::setContainer from service_arguments actions registration

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

As it would never be used

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]
:+1:

File: src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php [hunk 1]
```diff
@@ -65,11 +66,15 @@ public function process(ContainerBuilder $container)
             if (!$r = $container->getReflectionClass($class)) {
                 throw new InvalidArgumentException(sprintf('Class "%s" used for service "%s" cannot be found.', $class, $id));
             }
+            $isContainerAware = $r->implementsInterface(ContainerAwareInterface::class);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php]: missing `|| $r->isSubclassOf(AbstractController::class);`

Comment (chalasr) [inline src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php]: nice catch, fixed

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**